Client/Frontend/Sync/SyncWelcomeViewController.swift

```diff
@@ -238,7 +238,9 @@ class SyncWelcomeViewController: SyncViewController {
       }
 
       self.syncAPI.joinSyncGroup(codeWords: self.syncAPI.getSyncCode(), syncProfileService: self.syncProfileServices)
-      self.syncAPI.syncEnabled = true
+      self.syncAPI.requestSync()
+      self.syncAPI.setSetupComplete()
```

Comment:
what are those changes for?

Reply:
This is the change @Brandon-T made after `syncEnabled` is removed from Sync API.

Instead of setting syncEnabled value, we are requesting sync from sync worker and marking as setup complete.